src/main/java/unrc/dose/BellyService.java

```diff
@@ -0,0 +1,63 @@
+package unrc.dose;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+
+/** This is a Service to store a bunch of Bellies together. */
+public final class BellyService {
+
+    /** Bellies store. */
+    private static Map<String, Belly> bellies = new HashMap<String, Belly>();
+
+    /** Atomic counter. */
+    private static final AtomicInteger COUNT = new AtomicInteger(0);
+
+    /** Returns a particular belly.
+     * @param id String
+     * @return Belly
+    */
+    public Belly findById(final String id) {
+        return (Belly) bellies.get(id);
+    }
+
+    /** Adds a belly to store.
+     * @param name String
+     * @return Belly
+    */
+    public Belly add(final String name) {
+        int currentId = COUNT.incrementAndGet();
+        Belly belly = new Belly(currentId, name);
+        bellies.put(String.valueOf(currentId), belly);
+        return belly;
+    }
```

Comment:
There should be a check for the `name` argument, specifically checking for `null` and an empty value. Does adding several bellies with the same name should be considered a valid action? If not then there should be a check for that.